packages/create-emotion-styled/src/index.js

```diff
@@ -121,11 +121,14 @@ function createEmotionStyled(emotion: Emotion, view: ReactType) {
               styles.concat(classInterpolations)
             )
           } else {
-            className += staticClassName
+            className = `${staticClassName} ${className}`
           }
 
           if (stableClassName !== undefined) {
-            className += ` ${stableClassName}`
+            if (staticClassName === undefined) {
```

Comment:
I'll see why other line got changed, but could u explain this one? is it just to prevent unnecessary space being added? does it cause any harm?